### DISCUSSION TITLE
1.2 版本java udf 函数多参数报错问题

### DISCUSSION THREAD

**Discussion by fisher997** [apache/doris#13856] [RFC]
mysql> CREATE FUNCTION udf_test(string,int) RETURNS string  PROPERTIES (     "file"="file:///home/parallels/tools/doris_1.2/apache-doris-1.2.0-alpha-2a5d3db-jdk1.8-arm/jar/Fisher-1.0-SNAPSHOT.jar",     "symbol"="UDF.Demo",     "type"="JAVA_UDF" );
Query OK, 0 rows affected (0.04 sec)

mysql>
mysql>
mysql> select udf_test("DORIS",1) ;
ERROR 1105 (HY000): errCode = 2, detailMessage = UdfRuntimeException: Unable to call create UDF instance.
CAUSED BY: UdfRuntimeException: Unable to find evaluate function with the correct signature: UDF.Demo.evaluate(TEXT, INT)
UDF contains:
    public java.lang.String UDF.Demo.evaluate(java.lang.String,java.lang.Integer,java.lang.Integer)

UDF.Demo :
public class Demo extends UDF {
    public  String evaluate(String field , Integer a) {
        return field.substring(0, a) + StringUtils.repeat("*", field.length() - a);
    }

}

**Comment by LemonLiTree** [comment]
已修复
https://github.com/apache/doris/pull/13854
